Gulpfile.js

```diff
@@ -199,9 +214,10 @@ function getMatchingFiles(aFileArgs, aFileType)
     return lMatchingFiles;
 }
 
-function getAllTestFiles(aDirectory)
+function getAllTestFiles(aDirectory, aFilter)
 {
     const lFiles = [];
+    aFilter = aFilter || "test.js" 
```

Comment:
without this, there's a bug in the demo functionality that doesn't ACTUALLY find demo files.